pkg/controller/plan/kubevirt.go

```diff
@@ -1848,6 +1893,14 @@ func (r *KubeVirt) podVolumeMounts(vmVolumes []cnv.Volume, configMap *core.Confi
 				MountPath: "/opt",
 			},
 		)
+		if extraConfigMapExists {
+			mounts = append(mounts,
+				core.VolumeMount{
+					Name:      "extra-v2v-conf",
```

Comment:
how about pulling `extra-v2v-conf` which is used here 3 times to a const?

Reply:
the names of the volume and the config map don't have to align, but ok, it could be that the code analysis tool would complain about that so made it a const